### ISSUE TITLE
An exception occurred: MOVED 8402 X.X.X.X:6379 in lib/Credis/Client.php on line 1062

### ISSUE THREAD

**Issue by sajalaspl** [colinmollenhour/Cm_Cache_Backend_Redis#145]
We're using this library in our XenForo site, and we've set up an AWS Redis instance with multiple nodes (cluster). 

But we are getting the following error when trying to retrieve redis key values from AWS instance:

An exception occurred: MOVED 8402 X.X.X.X:6379 in lib/Credis/Client.php on line 1062

So seems that this library is not able to handle MOVED (ASK redirections)...??

**Comment by colinmollenhour** [comment]
That is correct, unfortunately this command is not recognized. Pull requests are always welcome, though!
